eng/scripts/Verify-And-Merge-PRs.ps1

```diff
@@ -8,15 +8,15 @@ param(
 
 $ReadyForMerge = $true
 $mergablePRs = @()
-$headers = @{ }
+$headers = @{ "Content-Type" = "text/json" }
 $RetryCount = 5
 
 if ($null -eq $ShouldMerge) {
   $ShouldMerge = $null -ne $AuthToken;
 }
 
 if ($AuthToken) {
-  $headers = @{
+  $headers += @{
     Authorization = "bearer $AuthToken"
```

Comment:
Does this work for merging objects?

Reply:
Yes it does from my local testing. I didn't test to see what happens if there is a conflict but the syntax here does work.

Reply:
From a quick test if the key already exists it fails, but that shouldn't be the case here.